### ISSUE TITLE
Scalar Plots for Land/Ice/etc. result in an Internal Server Error

### ISSUE THREAD

**Issue by GFDL-Eric** [jkrasting/dora#17]
To reproduce, go to https://dora.gfdl.noaa.gov/analysis/scalar?id=cmip6-1243 or a similar link for an alternate experiment, and submit a model component like ice or land (anything but Atmosphere). The result will be an Internal Server Error.

**Comment by jkrasting** [comment]
Thanks @GFDL-Eric - Can you please post a traceback of what happens inside the container as it is running?

**Comment by GFDL-Eric** [comment]
@jkrasting here is the traceback:

```
Traceback (most recent call last):
  File "/opt/conda/envs/env/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3803, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 138, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 165, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 5745, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 5753, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'netrad_toa'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/envs/env/lib/python3.9/site-packages/gunicorn/workers/sync.py", line 136, in handle
    self.handle_request(listener, req, client, addr)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/conda/envs/env/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/dora/scalar.py", line 110, in scalardiags
    dset = [x.build_netrad_toa() for x in dset]
  File "/dora/scalar.py", line 110, in <listcomp>
    dset = [x.build_netrad_toa() for x in dset]
  File "/opt/conda/envs/env/lib/python3.9/site-packages/gfdlvitals/extensions.py", line 264, in build_netrad_toa
    self["netrad_toa"].attrs = {
  File "/opt/conda/envs/env/lib/python3.9/site-packages/pandas/core/frame.py", line 3804, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/opt/conda/envs/env/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3805, in get_loc
    raise KeyError(key) from err
KeyError: 'netrad_toa'
```

**Comment by jkrasting** [comment]
Thanks @GFDL-Eric.  It looks to me like this issue stems a bug that cropped up in gfdlvitals that was addressed by this commit: 
 https://github.com/jkrasting/gfdlvitals/pull/25/commits/25fe9f5e3b48f3d8ba97ccee0addd3824dfec839

This should be fixed in gfdlvitals v3.0.4.  Can you check the version that is triggering the error?

**Comment by jkrasting** [comment]
Hi @GFDL-Eric -- it turns out this patch did not make it into v3.0.4.  There is a new version out there on PyPi now (v3.0.5) that addresses this error.
